Source/Scene/Cesium3DTileset.js

```diff
@@ -496,6 +519,63 @@ define([
         }).otherwise(function(error) {
             that._readyPromise.reject(error);
         });
+
+        /**
+         * Determines if level-of-detail skipping optimization should be used.
+         *
+         * @type {Boolean}
+         * @default true
+         */
+        this.skipLODs = defaultValue(options.skipLODs, true);
+
+        /**
+         * This is a multiplier defining the minumum screen space error to skip.
+         * If current tile has screen space error of 100, no tiles will be loaded unless they
+         * are leaves or have a screen space error <= 100 / skipSSEFactor.
+         *
+         * Only used when tileset.skipLODs === true.
+         *
+         * @type {Number}
+         * @default 10
+         */
+        this._skipSSEFactor = defaultValue(options.skipSSEFactor, 10);
+
+        /**
+         * This is a constant defining the minumum number of levels skip.
+         * If current tile is level 1, no tiles will be loaded unless they
+         * are at level greater than 2.
+         *
+         * Only used when tileset.skipLODs === true.
+         *
+         * @type {Number}
+         * @default 1
+         */
+        this._skipLevels = defaultValue(options.skipLevels, 1);
+
+        /**
+         * Determines whether low memory level-of-detail skipping should be used.
+         * When true, only tiles that meet the maximum screen space error will ever be downloaded.
+         *
+         * Only used when tileset.skipLODs === true.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.lowMemory = defaultValue(options.lowMemory, false);
+
+        /**
+         * Determines whether sibling tiles should be loaded when skipping levels-of-detail.
+         * When true, the siblings of any visible and downloaded tile are downloaded as well.
+         *
+         * Only used when tileset.skipLODs === true.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.loadSiblings = defaultValue(options.loadSiblings, false);
+
+        this._loadHeaps = {};
+        this._refining = false;
```

Comment:
I don't think so. `_refining` should probably be renamed to something like `_hasMixedContent`. It only gets set to true when we traverse up the tree to select an ancestor, resulting in mixed LODs. There can be cases when there are tiles that need to be loaded, but there's no mixing of LODs.

For example, if you start close to a tileset with a high skipping factor and zoom out, no ancestors will be loaded, so those will queued for load, but the existing tiles that are loaded will remain selected because there's nothing else (we're selecting descendants of the desired tiles because nothing else is ready). In this case, the tileset won't `_hasMixedContent`, but there will still be tiles to load.